console/frontend/src/main/frontend/src/app/views/login/login.component.ts

```diff
@@ -24,19 +25,26 @@ export class LoginComponent implements OnInit {
   constructor(
     private alertService: AlertService,
     private authService: AuthService,
+    private router: Router,
   ) {}
 
   ngOnInit(): void {
-    this.authService.loggedin(); // Check whether the client is logged in.
+    if (this.authService.isLoggedIn()) {
+      this.router.navigate(['/']);
+    }
 
     window.setTimeout(() => {
```

Comment:
Couldn't this timeout be substituted with ngAfterViewInit?

Reply:
I guess, I'm not fully sure on what the intent was originally
I'll update it